packages/components/src/lozenge/docs/Lozenge.stories.mdx

```diff
@@ -89,6 +88,16 @@ A lozenge can be highlighted to bring more attention to it when required. This v
     </Story>
 </Preview>
 
+### Fluid
+
+A lozenge can take all the available space.
+
+<Preview>
+    <Story name="fluid">
+        <Lozenge fluid>NEW</Lozenge>
+    </Story>
+</Preview>
+
```

Comment:
Can we get a quick screenshot of what it looks like in Storybook. I would add this to the PR description everytime a change is made just for a quick overview of what changed without having to pull the branch

Reply:
![image](https://user-images.githubusercontent.com/361632/180065078-45103191-5532-431f-823d-0cdeeeec5985.png)

you can look at the preview ;)

Reply:
Nevermind, the links from netlify do this